plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterCheckboxList.php

```diff
@@ -57,7 +57,7 @@ function ( $item ) {
 			'data-wp-key'         => wp_unique_prefixed_id( $this->get_full_block_name() ),
 			'data-wp-context'     => '{}',
 			'class'               => esc_attr( $classes ),
-			'style'               => esc_attr( $style ),
+			'style'               => esc_attr( $style ) . ';',
```

Comment:
Could you clarify the reason for this change? Currently, when style is empty, the generated HTML looks like this:

```
<div data-block-name="woocommerce/product-filter-checkbox-list" style=";" class="wp-block-woocommerce-product-filter-checkbox-list wc-block-product-filter-checkbox-list wp-block-woocommerce-product-filter-checkbox-list" data-wp-interactive="woocommerce/product-filters" data-wp-key="woocommerce/product-filter-checkbox-list4" data-wp-context="{}">
```

Reply:
Ah, I should check if the style is empty or not, will update it.

The reason for that is, the generated style by Supports API doesn't have the `;` at the end. When passing it to `get_block_wrapper_attributes()`, we need to append the `;`, otherwise, the style will break.

Reply:
Updated in [12843ff](https://github.com/woocommerce/woocommerce/pull/58517/commits/12843ff01568d8c59626917e41360befa777114a)

Reply:
Ah it makes sense! Thanks! 🙇